modules/scripts/startup-script/main.tf

```diff
@@ -93,6 +93,20 @@ locals {
     }
   ]
 
+  ofi_runner = var.set_ofi_cloud_rdma_tunables == "" ? [] : [
+    {
+      type        = "data"
+      destination = "/etc/profile.d/set_ofi_cloud_rdma_tunables.sh"
+      content     = <<-EOT
+        #!/bin/bash
+        export FI_PROVIDER="verbs;ofi_rxm"
+        export FI_OFI_RXM_USE_RNDV_WRITE=1
+        export FI_VERBS_INLINE_SIZE=39
+        export I_MPI_FABRICS="shm:rxm"
```

Comment:
The `export` command only makes the variable available to child shells/processes, so it won't be set when the user logs in.

Reply:
My understanding is since this is placed as a `profile.d` and runs a start up script as long as a user logs in after start up scripts complete it will be there. If they log in prior then they would need to source their .bashrc again

Reply:
Ahh I see, missed that it was in `/etc/profile.d`.  Thanks for pointing that out!